env.template

```diff
@@ -1,4 +1,8 @@
-# Copy this file to both ".env.dev" and ".env.docker.dev" & tweak values for local development
+# Copy this file to ".env.dev" and tweak values for local development
 FREQUENCY_URL=ws://0.0.0.0:9944
-REDIS_HOST=localhost
+REDIS_URL=redis://localhost
 REDIS_PORT=6379
```

Comment:
REDIS_PORT can get baked into the url like so:
`REDIS_URL=redis://localhost:6379`